ERROR - 2025-08-24 05:06:01 --> mysqli_sql_exception: Table 'lms_latangga.users' doesn't exist in C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', NULL, false)
#4 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-LATANGGA\app\Database\Seeds\UserSeeder.php(33): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\Seeder.php(149): App\Database\Seeds\UserSeeder->run()
#7 C:\xampp\htdocs\ITE311-LATANGGA\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-LATANGGA\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-24 05:22:17 --> mysqli_sql_exception: Table 'lms_latangga.users' doesn't exist in C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', NULL, false)
#4 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-LATANGGA\app\Database\Seeds\UserSeeder.php(33): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\Seeder.php(149): App\Database\Seeds\UserSeeder->run()
#7 C:\xampp\htdocs\ITE311-LATANGGA\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-LATANGGA\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-24 05:37:01 --> mysqli_sql_exception: Table 'lms_latangga.users' doesn't exist in C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', NULL, false)
#4 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-LATANGGA\app\Database\Seeds\UsersSeeder.php(55): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\Seeder.php(149): App\Database\Seeds\UsersSeeder->run()
#7 C:\xampp\htdocs\ITE311-LATANGGA\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-LATANGGA\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-24 05:37:25 --> mysqli_sql_exception: Table 'lms_latangga.users' doesn't exist in C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', NULL, false)
#4 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-LATANGGA\app\Database\Seeds\UsersSeeder.php(55): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\Seeder.php(149): App\Database\Seeds\UsersSeeder->run()
#7 C:\xampp\htdocs\ITE311-LATANGGA\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-LATANGGA\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-24 05:37:34 --> mysqli_sql_exception: Table 'lms_latangga.users' doesn't exist in C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', NULL, false)
#4 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-LATANGGA\app\Database\Seeds\UsersSeeder.php(55): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\Seeder.php(149): App\Database\Seeds\UsersSeeder->run()
#7 C:\xampp\htdocs\ITE311-LATANGGA\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-LATANGGA\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-24 05:39:28 --> mysqli_sql_exception: Can't create table `lms_latangga`.`submissions` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `s...', 0)
#1 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `s...')
#2 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `s...')
#3 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `s...')
#4 C:\xampp\htdocs\ITE311-LATANGGA\app\Database\Migrations\2025-08-24-045851_CreateSubmissionsTable.php(46): CodeIgniter\Database\Forge->createTable('submissions')
#5 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->up()
#6 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-LATANGGA\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-LATANGGA\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-24 05:39:55 --> mysqli_sql_exception: Can't create table `lms_latangga`.`submissions` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `s...', 0)
#1 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `s...')
#2 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `s...')
#3 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `s...')
#4 C:\xampp\htdocs\ITE311-LATANGGA\app\Database\Migrations\2025-08-24-045851_CreateSubmissionsTable.php(46): CodeIgniter\Database\Forge->createTable('submissions')
#5 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->up()
#6 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-LATANGGA\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-LATANGGA\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-24 05:43:16 --> mysqli_sql_exception: Can't create table `lms_latangga`.`submissions` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `s...', 0)
#1 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `s...')
#2 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `s...')
#3 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `s...')
#4 C:\xampp\htdocs\ITE311-LATANGGA\app\Database\Migrations\2025-08-24-045851_CreateSubmissionsTable.php(46): CodeIgniter\Database\Forge->createTable('submissions')
#5 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->up()
#6 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-LATANGGA\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-LATANGGA\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-24 05:43:50 --> mysqli_sql_exception: Can't create table `lms_latangga`.`submissions` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `s...', 0)
#1 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `s...')
#2 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `s...')
#3 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `s...')
#4 C:\xampp\htdocs\ITE311-LATANGGA\app\Database\Migrations\2025-08-24-045851_CreateSubmissionsTable.php(46): CodeIgniter\Database\Forge->createTable('submissions')
#5 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->up()
#6 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-LATANGGA\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-LATANGGA\system\Commands\Database\MigrateRefresh.php(87): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\ITE311-LATANGGA\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2025-08-24 05:45:21 --> mysqli_sql_exception: Can't create table `lms_latangga`.`submissions` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `s...', 0)
#1 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `s...')
#2 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `s...')
#3 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `s...')
#4 C:\xampp\htdocs\ITE311-LATANGGA\app\Database\Migrations\2025-08-24-045851_CreateSubmissionsTable.php(46): CodeIgniter\Database\Forge->createTable('submissions')
#5 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->up()
#6 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-LATANGGA\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-LATANGGA\system\Commands\Database\MigrateRefresh.php(87): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\ITE311-LATANGGA\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2025-08-24 05:48:10 --> mysqli_sql_exception: Can't create table `lms_latangga`.`submissions` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `s...', 0)
#1 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `s...')
#2 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `s...')
#3 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `s...')
#4 C:\xampp\htdocs\ITE311-LATANGGA\app\Database\Migrations\2025-08-24-045851_CreateSubmissionsTable.php(46): CodeIgniter\Database\Forge->createTable('submissions')
#5 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->up()
#6 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-LATANGGA\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-LATANGGA\system\Commands\Database\MigrateRefresh.php(87): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\ITE311-LATANGGA\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2025-08-24 05:48:32 --> mysqli_sql_exception: Can't create table `lms_latangga`.`submissions` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `s...', 0)
#1 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `s...')
#2 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `s...')
#3 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `s...')
#4 C:\xampp\htdocs\ITE311-LATANGGA\app\Database\Migrations\2025-08-24-045851_CreateSubmissionsTable.php(45): CodeIgniter\Database\Forge->createTable('submissions', true)
#5 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->up()
#6 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-LATANGGA\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-LATANGGA\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-24 05:55:11 --> mysqli_sql_exception: Unknown column 'username' in 'field list' in C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', NULL, false)
#4 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-LATANGGA\app\Database\Seeds\UsersSeeder.php(55): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-LATANGGA\system\Database\Seeder.php(149): App\Database\Seeds\UsersSeeder->run()
#7 C:\xampp\htdocs\ITE311-LATANGGA\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-LATANGGA\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-LATANGGA\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-LATANGGA\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
